src/main/java/frc/robot/commands/Claw/AutoGripperInCmd.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.commands.Claw;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.GripperSubsystem;
+
+public class AutoGripperInCmd extends CommandBase {
+  private Timer timer = new Timer();
+  /** Creates a new RollerPush. */
+  GripperSubsystem gripper;
+
+  public AutoGripperInCmd(GripperSubsystem grippers) {
+    gripper = grippers;
+    addRequirements(grippers);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    timer.reset();
+    timer.start();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    gripper.rotateinGripper();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    gripper.stopGripper();
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
```

Comment:
This function can be rewritten for simplicity like:

```java
public boolean isFinished() {
  if (timer.get() > 1.5) {
    return true;
  }
  return false;
}
```

Setting the values to variables first can be helpful if we want to print them or send them to the smartdashboard.  If not, it can be condensed for readability.